### PR TITLE
Add  submitted date to Export RIP-247

### PR DESCRIPTION
**Pull request by autotelik**
https://eaflood.atlassian.net/browse/RIP-247

Add submitted_at to Enrollment and set when user ends journey.

### CODE REVIEW

**Comment by autotelik** [comment]
:eyes:

File: app/models/flood_risk_engine/enrollment.rb [hunk 1]
```diff
@@ -78,6 +78,12 @@ def state_machine
       withdrawn: 6        # BO: used to hide anything created in error
     }
 
+    def submit
+      return if submitted_at.present?
+      self.submitted_at = Time.zone.now
+      pending!
```
Comment (reggieb) [inline app/models/flood_risk_engine/enrollment.rb]: Should the `pending!` method be called on submission even when there is an existing `submitted_at`?

**Comment by reggieb** [comment]
:ship: subject to addressing the `pending!` issue.

File: app/models/flood_risk_engine/enrollment.rb (hunk 1 above)
Comment (autotelik) [inline app/models/flood_risk_engine/enrollment.rb]: I don't believe so, this should be a one off, and if pending needs setting in other circumstances can be fired individually - AFAIK this check wanted, to stop some issues found in WEX with states getting reset, when NCCC started editing enrollments

Comment (reggieb) [inline app/models/flood_risk_engine/enrollment.rb]: OK - Just thought I'd check.

**Pull request merged**